python/packages/autogen-ext/src/autogen_ext/models/anthropic/_anthropic_client.py

```diff
@@ -160,7 +160,7 @@ def user_message_to_anthropic(message: UserMessage) -> MessageParam:
                 blocks.append(
                     ImageBlockParam(
                         type="image",
-                        source=Source(
+                        source=Base64ImageSourceParam(
```

Comment:
Thanks @SongChiYoung , this looks good.
The final piece is that you will need to run `uv sync --all-extras` to update the uv.lock file (to ensure the dependency updates for the anthropic client are added).

Once it is updated, you can then commit file (yes, commit the uv.lock file). 

Once that is done, we'll merge this in!

Thanks again!